src/SimpleS3.php

```diff
@@ -101,7 +95,7 @@ private function s3Request(string $httpVerb, string $bucket, string $key, array
         } else {
             $url = "https://$hostname";
         }
-        $url = "$url{$uriPath}?$queryString";
+        $url = "$url$uriPath?$queryString";
```

Comment:
I'd rather keep this one as it was because it was more readable.

Reply:
OK will revert the changes

Reply:
Reverted in the last commit